data/events.yaml

```diff
@@ -0,0 +1,610 @@
+x-events:
+  - name: generic_cluster
+    message: "{message}"
+    event_type: cluster
+    severity: "info"
+    properties:
+      cluster_id: UUID
+      message: string
+      severity: string
+
+  - name: generic_host
+    message: "{message}"
+    event_type: host
+    severity: "info"
+    properties:
+      cluster_id: UUID
+      host_id: UUID
+      message: string
+      severity: string
+
+  - name: cancel_install_failed_start
+    message: "Failed to cancel installation: error starting DB transaction"
+    event_type: cluster
+    severity: "error"
+    properties:
+        cluster_id:  UUID
+
+  - name: cancel_install_failed_commit
+    message: "Failed to cancel installation: error committing DB transaction"
+    event_type: cluster
+    severity: "error"
+    properties:
+        cluster_id:  UUID
+
+  - name: iso_gen_failed_format_ign
+    message: "Failed to generate image: error formatting ignition file"
+    event_type: cluster
+    severity: "error"
+    properties:
+        cluster_id:  UUID
+
+  - name: iso_gen_failed
+    message: "Failed to generate minimal ISO"
+    log_message: "Failed to generate minimal ISO for cluster {cluster_id}"
+    event_type: cluster
+    severity: "error"
+    properties:
+        cluster_id:  UUID
+
+  - name: cluster_discovery_ignition_config_applied
+    message: "Custom discovery ignition config was applied to the cluster"
+    log_message: "Custom discovery ignition config was applied to cluster {cluster_id}"
+    event_type: cluster
+    severity: "info"
+    properties:
+      cluster_id:  UUID
+
+  - name: backend_image_deleted
+    message: "Deleted image from backend because its ignition was updated. The image may be regenerated at any time."
+    event_type: cluster
+    severity: "info"
+    properties:
+      cluster_id:  UUID
+
+  - name: successful_cluster_registration
+    message: "Successfully registered to the cluster"
+    log_message: "Successfully registered cluster {cluster_name} with id {cluster_id}"
```

Comment:
let's unify `log_message` and `message` into `message`
if both exists, prefer the value of  `log_message` for the `message` attribute

Reply:
solved